src/DataMigration/docs/New-AzDataMigrationToSqlManagedInstance.md

```diff
@@ -0,0 +1,563 @@
+---
+external help file:
+Module Name: Az.DataMigration
+online version: https://docs.microsoft.com/powershell/module/az.datamigration/new-azdatamigrationtosqlmanagedinstance
+schema: 2.0.0
+---
+
+# New-AzDataMigrationToSqlManagedInstance
+
+## SYNOPSIS
+Create or Update Database Migration resource.
+
+## SYNTAX
+
+```
+New-AzDataMigrationToSqlManagedInstance -ManagedInstanceName <String> -ResourceGroupName <String>
+ -TargetDbName <String> [-SubscriptionId <String>] [-AzureBlobAccountKey <String>]
+ [-AzureBlobContainerName <String>] [-AzureBlobStorageAccountResourceId <String>]
+ [-FileSharePassword <String>] [-FileSharePath <String>] [-FileShareUsername <String>] [-Kind <ResourceType>]
+ [-MigrationOperationId <String>] [-MigrationService <String>] [-OfflineConfigurationLastBackupName <String>]
+ [-OfflineConfigurationOffline] [-ProvisioningError <String>] [-Scope <String>] [-SourceDatabaseName <String>]
+ [-SourceSqlConnectionAuthentication <String>] [-SourceSqlConnectionDataSource <String>]
+ [-SourceSqlConnectionEncryptConnection] [-SourceSqlConnectionPassword <String>]
+ [-SourceSqlConnectionTrustServerCertificate] [-SourceSqlConnectionUserName <String>]
+ [-TargetDatabaseCollation <String>] [-TargetLocationAccountKey <String>]
+ [-TargetLocationStorageAccountResourceId <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait]
+ [-PassThru] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create or Update Database Migration resource.
+
+## EXAMPLES
+
+### Example 1: Start a Database Migration from the on-premise Source Sql Server to target Managed Instance
+```powershell
+PS C:\> New-AzDataMigrationToSqlManagedInstance -ResourceGroupName "MyResourceGroup" -ManagedInstanceName "MyManagedInstance" -TargetDbName "MyDb" -Kind "SqlMI" -Scope "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyResourceGroup/providers/Microsoft.Sql/managedInstances/MyManagedInstance" -MigrationService "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyRG/providers/Microsoft.DataMigration/SqlMigrationServices/MySqlMigrationService" -TargetLocationStorageAccountResourceId "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyResourceGroup/providers/Microsoft.Storage/storageAccounts/MyStorageAccount" -TargetLocationAccountKey "dwidjwiwnnwiojnnmskoam==adnkdnwdnwknk" -FileSharePath "\\filesharepath.com\SharedBackup\MyBackUps" -FileShareUsername "filesharepath\User" -FileSharePassword "password" -SourceSqlConnectionAuthentication "SqlAuthentication" -SourceSqlConnectionDataSource "LabServer.database.net" -SourceSqlConnectionUserName "User" -SourceSqlConnectionPassword "password" -SourceDatabaseName "AdventureWorks"
```

Comment:
Please use the fake subscripiton

Reply:
These are fake subscription, but I will update it to 0000-1111-2222-3333 to avoid future confusion.

Reply:
Updated